inc/class-plugin.php

```diff
@@ -253,7 +253,7 @@ public function get_s3_bucket_region() : ?string {
 	public function get_original_upload_dir() : array {
 
 		if ( empty( $this->original_upload_dir ) ) {
-			wp_upload_dir();
+			return wp_upload_dir();
```

Comment:
@mitogh this was intentional. Calling `wp_upload_dir` should always set `$this->original_upload_dir`. Otherwise doing this will return the `s3://` dir potentially.

Reply:
Oh makes sense, I was under the assumption this was a default value not a setup helper, thanks for the context :)